docs/framework/configure-apps/file-schema/startup/supportedruntime-element.md

```diff
@@ -5,14 +5,13 @@ ms.prod: ".net-framework"
 ms.technology: 
   - "dotnet-clr"
 ms.topic: "article"
+ms.custom: "updateeachrelease"
 f1_keywords: 
   - "http://schemas.microsoft.com/.NetConfiguration/v2.0#supportedRuntime"
   - "http://schemas.microsoft.com/.NetConfiguration/v2.0#configuration/startup/supportedRuntime"
 helpviewer_keywords: 
   - "supportedRuntime element"
   - "<supportedRuntime> element"
-ms.assetid: 1ae16e23-afbe-4de4-b413-bc457f37b69f
```

Comment:
for topics that were migrated from caps, I'm keeping this optional field (for now at least). It helps me find the old version of it more easily when there are issues (and unfortunately, we're still finding issues). I'm fine with removing the revision number from the topics though.